src/webview/components/memory-table.tsx

```diff
@@ -17,14 +17,18 @@
 import { DebugProtocol } from '@vscode/debugprotocol';
 import memoize from 'memoize-one';
 import { Column } from 'primereact/column';
-import { DataTable, DataTableCellSelection, DataTableProps, DataTableRowData, DataTableSelectionCellChangeEvent } from 'primereact/datatable';
+import { DataTable, DataTableCellSelection, DataTableColumnResizeEndEvent, DataTableProps, DataTableRowData, DataTableSelectionCellChangeEvent } from 'primereact/datatable';
 import { ProgressSpinner } from 'primereact/progressspinner';
 import React from 'react';
 import { TableRenderOptions } from '../columns/column-contribution-service';
 import { Decoration, Memory, MemoryDisplayConfiguration, ScrollingBehavior, isTrigger } from '../utils/view-types';
 import isDeepEqual from 'fast-deep-equal';
 import { AddressColumn } from '../columns/address-column';
 import { classNames } from 'primereact/utils';
+import { CONFIG_GROUPS_PER_ROW_NUMERIC_CHOICES } from '../../plugin/manifest';
+import { isNumber } from '../../common/typescript';
+import { characterWidthInContainer, elementInnerWidth, fontSize } from '../utils/window';
+import { DataColumn } from '../columns/data-column';
```

Comment:
Is there a way that we could make different fitting behaviors part of the column API so that we wouldn't have hardcodings for the `AddressColumn` and `DataColumn` here?